config/pmd-ruleset.xml

```diff
@@ -26,10 +26,13 @@
         <exclude name="JUnitUseExpected" /> <!-- Favor using fail explicitly to test correct exception -->
```

Comment:
I'm not going to both inspecting these changes as most code will be Kotlin going forward anyway.